### PR TITLE
README: Cleanup Working on the Exercises

### LINKED ISSUE #407: Python version support: 3.3 or 3.5?
The [README](https://github.com/exercism/xpython/blob/master/README.md) states: 

> All exercises must be compatible with Python versions 2.7 and 3.3 upwards. Therefore please test your changes at least with Python2.7 and Python3.5.

I'm confused. Should we test with Python**3.3** or Python**3.5**? If all exercises must be compatible with 3.3 upwards should we instruct users to test with Python3.3 instead?

### PR DESCRIPTION
**Pull request by rootulp**
Closes #407

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +15,19 @@ We welcome both improvements to the existing exercises and new exercises.
 A pool of exercise ideas can be found in the [x-common repo](https://github.com/exercism/x-common).
 
 All exercises must be compatible with Python versions 2.7 and 3.3 upwards.
-Therefore please test your changes at least with Python2.7 and Python3.5.
+Therefore please test your changes with these versions.
 
-To test a single exercise, say crypto-square, run:
+Test a single exercise with Python 2.7:
 ```
 python2.7 test/check-exercises.py crypto-square
```
Comment (behrtam) [inline README.md]: Should we make this more generic now that you dropped the "say crypto-square" part?
Maybe: `python2.7 test/check-exercises.py [exercise-name]`

File: README.md [hunk 2]
```diff
@@ -15,18 +15,19 @@ We welcome both improvements to the existing exercises and new exercises.
...
 ```
-and
+
+Test a single exercise with Python 3.3:
 ```
-python3.5 test/check-exercises.py crypto-square
+python3.3 test/check-exercises.py crypto-square
```
Comment (behrtam) [inline README.md]: Do we really need the python2 and python3 call example?
We have the phrase "please test your changes with these versions" so people know what to test and than one single exercise example and that for all should be enough, right?

File: README.md (hunk 1 above)
Comment (rootulp) [inline README.md]: Agreed. Will fix.

File: README.md (hunk 2 above)
Comment (rootulp) [inline README.md]: If we expect contributors to test with both versions, I think we should list both command examples. I don't feel too strongly on this. If you think we can trust contributors to perform the tests themselves, we can remove this command example.

**Pull request merged**